sched/scheduler/jobStatus_test.go

```diff
@@ -0,0 +1,171 @@
+package scheduler
+
+import (
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/prop"
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+	"testing"
+)
+
+func Test_GetJobStatus_Error(t *testing.T) {
```

Comment:
We should really have that meeting about testing methodology. I don't care if it's fake or mock but I would want to refactor these to look more like Dan's latest PR (it's what I'm more comfortable with given shared ownership of [test] code).